src/index.js

```diff
@@ -69,8 +69,18 @@ const pactum = {
     return klona(value);
   },
 
-  parse
+  parse,
 
+  getStoreKey: stash.getStoreKey,
```

Comment:
I don't think these are no longer required here. User can import respective functions and constants from `stash`